lib/index.js

```diff
@@ -4,10 +4,10 @@
 "use strict";
 
 var path  = require('path'),
-    zip   = require('jszip'),
     sizeOf = require('image-size'),
     fs = require('fs'),
     etree = require('elementtree');
+import zip from "jszip";
```

Comment:
@kant2002 I am using node 12.14.1 and I am receiving the following error:
```
import zip from "jszip";
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

Is there any specific reason to change the previous `require` with `import`?

Reply:
Hi @kant2002

Currently using latest version 1.3.1 catch this error, and also have same question why we have import ... from ... instead of require?

@fslc92 have you solution about this?

Thank you,

Reply:
I believe I fixed that in 1.3.2. Sorry. If that does not fix an issue, can you give me small repro?

Reply:
Thank you @kant2002, it's work in my local and my server, Thank you.
I think time goes to sleep then :D

Reply:
It is day in the shiny Kazakhstan right now, so probably I have to work.

Reply:
Glad to have support from you 🍻  My team thinks this repo is out of maintenance, the last commit in 2 years ago 😅

Reply:
I do maintain, but I outside of JS ecosystem right now. Most imprevements comes from users right now. Obviously this library require some love. I do not want put a banner "I do not develop" because I do not develop, but other developers usually are. This will scary people.

Sometimes I just forget to comeback to repo. Since I support code + tests, that's usually enough to not break users.